async_service/base.py

```diff
@@ -7,7 +7,27 @@
 
 
 class Service(ServiceAPI):
-    pass
+    @property
+    def manager(self) -> "InternalManagerAPI":
+        """
+        Expose the manager as a property here intead of
+        :class:`async_service.abc.ServiceAPI` to ensure that anyone using
+        proper type hints will not have access to this property since it isn't
+        part of that API, while still allowing all subclasses of the
+        :class:`async_service.base.Service` to access this property directly.
+
+        This allows for
```

Comment:
This allows for?

Reply:
will fill out, looks like I got distracted mid-doc-string.